app/models/transformation_mapping.rb

```diff
@@ -82,18 +83,23 @@ def describe_non_vm(vm_name)
   def describe_vm(vm, reason)
     {
       "name"           => vm.name,
-      "cluster"        => vm.ems_cluster.name,
-      "path"           => "#{vm.ext_management_system.name}/#{vm.parent_blue_folder_path(:exclude_non_display_folders => true)}",
+      "cluster"        => vm.ems_cluster.try(:name) || '',
+      "path"           => vm.ext_management_system ? "#{vm.ext_management_system.name}/#{vm.parent_blue_folder_path(:exclude_non_display_folders => true)}" : '',
       "allocated_size" => vm.allocated_disk_storage,
       "id"             => vm.id,
       "reason"         => reason
     }
   end
 
   def validate_vm(vm, quick = true)
+    validate_result = vm.validate_v2v_migration
+    return validate_result unless validate_result == VM_VALID
+
     # a valid vm must find all resources in the mapping and has never been migrated
     invalid_list = []
 
+    return VM_INACTIVE unless vm.active?
```

Comment:
remove this line `return VM_INACTIVE unless vm.active?`. Looks good else.

Reply:
Yes, agree that should have been removed. Thanks